docker-hub/webhooks.md

```diff
@@ -6,14 +6,14 @@ title: Webhooks for automated builds
 
 If you have an automated build repository in Docker Hub, you can use Webhooks to
 cause an action in another application in response to an event in the
-repository. Docker Hub webhooks fire when an image is built in, or a new tag
+repository. Webhook is a POST request sent to a defined URL which provides the service. Docker Hub webhooks fire when an image is built in, or a new tag
 is added to, your automated build repository.
 
 Configure webhooks on `https://hub.docker.com/r/<USERNAME>/<REPOSITORY>/~/settings/webhooks/`.
 
 ![Create Webhook](/docker-hub/images/webhooks.png)
 
-With your webhook, you specify a target URL to POST to. Docker Hub POSTs
+With your webhook, you specify a target URL to POST to. URL that accepts data from the Webhook, stores this POST request in JSON or XML format. Docker Hub POSTs
```

Comment:
With the change you've made to lines 7-9 this whole line feels a bit redundant, so I think we can remove the whole thing and just leave something like "Docker Hub sends a POST to your URL, containing the following payload:"

What do you think?